kentik_api_library/examples/queries_example.py

```diff
@@ -115,7 +117,11 @@ def run_query_chart() -> None:
     result = client.query.chart(query_object)
 
     print("Result:")
-    print(result.__dict__)
+    # result.save_image_as("/home/lukasz/test.png")
+    img = Image.open(BytesIO(result.get_data()))
+    img.show()
+    print("File saved")
```

Comment:
not sure if it is actually saved?

Reply:
You are right it is not saved but i forgot to remove this print